libminifi/include/core/state/nodes/MetricsBase.h

```diff
@@ -37,28 +37,28 @@ namespace org::apache::nifi::minifi::state::response {
 class ResponseNode : public core::Connectable, public PublishedMetricProvider {
  public:
   ResponseNode()
-      : core::Connectable("metric"),
-        is_array_(false) {
+    : core::Connectable("metric"),
+      is_array_(false) {
   }
 
   explicit ResponseNode(std::string name)
-      : core::Connectable(std::move(name)),
-        is_array_(false) {
+    : core::Connectable(std::move(name)),
+      is_array_(false) {
   }
 
   ResponseNode(std::string name, const utils::Identifier& uuid)
-      : core::Connectable(std::move(name), uuid),
-        is_array_(false) {
+    : core::Connectable(std::move(name), uuid),
+      is_array_(false) {
   }
 
   ~ResponseNode() override = default;
 
-  static std::vector<state::response::SerializedResponseNode> serializeAndMergeResponseNodes(const std::vector<std::shared_ptr<ResponseNode>>& nodes) {
+  static std::vector<SerializedResponseNode> serializeAndMergeResponseNodes(const std::vector<gsl::not_null<std::shared_ptr<ResponseNode>>>& nodes) {
```

Comment:
should we use the `SharedResponseNode` alias here as well?

Reply:
Updated in c71bdaa3378b7087a33b7bd5f9a6bf4c3dd8cff8